apple/Sources/Sargon/Extensions/Methods/Prelude/Decimal192+Wrap+Functions.swift

```diff
@@ -73,6 +73,17 @@ extension Decimal192 {
 	public static let maxDivisibility: UInt8 = 18
     
     public static let temporaryStandardFee: Self = transactionFeePreset()
+	
+	public static let one: Self 	= 1
```

Comment:
I doubt about it myself as well. The `+Swiftified` file makes more sense, but shouldn't `maxDivisibility` & `temporaryStandardFee` be there as well then?

Reply:
no because `transactionFeePreset()` is calling a global method.

Reply:
and `maxDivisibility` IS declared in Rust, but was not able to UniFFI export it, since one cannot UniFFI export static variables

Reply:
tbh, I have been wondering for a while whether we shouldn't have both extensions in same file. There are also cases where we use global methods from the Swiftified extension ([example](https://github.com/radixdlt/sargon/blob/main/apple/Sources/Sargon/Extensions/Swiftified/Prelude/Decimal192%2BSwiftified.swift#L87)), and it could be easier to maintain to keep all in one place (and have only the `+SampleValues` extension in a different file).

Reply:
The split is much wanted! The goal is to be able to delete the whole `+Wrap+Address`!  It is needed because UniFFI **currently** does not allow for exports of constructors for `[rust] Record` (`[swift] struct`).

When they do, we can remove many wrapped functions, e.g
https://github.com/radixdlt/sargon/blob/main/apple/Sources/Sargon/Extensions/Methods/Prelude/Decimal192%2BWrap%2BFunctions.swift#L11